operator/test/test.sh

```diff
@@ -160,6 +214,11 @@ test_chains() {
     echo "[ERROR] Public key is not accessible" >&2
     exit 1
   fi
+
+  echo -n "  - Check Pod Security tekton-chains: "
```

Comment:
I would argue that `test_chains` should test the chains functionality from a user point of view. My preference would be to create a new set of test for security.

Reply:
ok

i will reorganize